docs/about/intro.adoc

```diff
@@ -27,12 +27,12 @@
 
 * Introduction of the new Helidon WebServer (Project N&iacute;ma): a virtual threads-based web server implementation based on https://openjdk.org/jeps/444[JDK Project Loom]  virtual threads.
 
-* Removed Helidon's Reactive WebServer and WebClient that were based on Netty. Our new implementations are based on virtual threads that have a blocking style API (Helidon N&iacute;ma). Learn more: <<Helidon 4 WebServer, Helidon 4 WebServer.>>
+* Removed Helidon's Reactive WebServer and WebClient that were based on Netty. Our new implementations are based on virtual threads that have a blocking style API (Project N&iacute;ma). Learn more: <<Helidon 4 WebServer, Helidon 4 WebServer.>>
```

Comment:
Maybe: previously known as "Project Níma" ?

Reply:
I discussed this with Joe, Ed and Dmitry and they requested I use Project Nima just in this intro and then refer to it as the Helidon WebServer everywhere else.

Reply:
Ok